src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/CSharpProjectFile.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
```

Comment:
Please add copyright headers to all code files.

```c#
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

...
```

Reply:
# Resolved